webrtc.html

```diff
@@ -1031,6 +1031,20 @@ <h4>Operation</h4>
         representation of an <code><a>RTCIceTransport</a></code> changes, as
         described in <a href= "#rtcicetransport"></a>.</p>
 
+        <p data-link-for="RTCPeerConnection">The state of the SDP
+          negotiation is represented by
+        the <a>connection state</a> and the internal
+        variables <a>[[\CurrentLocalDescription]]</a>,
+        <a>[[\CurrentRemoteDescription]]</a>, <a>[[\PendingLocalDescription]]</a>
+        and <a>[[\PendingRemoteDescription]]</a>. These are only set
+        inside the <code><a>setLocalDescription</a></code>
+        and <code><a>setRemoteDescription</a></code> operations, and
+        modified by the <code><a>addIceCandidate</a></code> operation and
+        the <a>surface a candidate</a> procedure. In each case, all
+        the modifications to all the five variables are completed
+        before the procedures fire any events or invoke any callbacks,
+        so the modifications are made visible as a single event.</p>
```

Comment:
Considering we use internal slots, and are clear about when these are updated, I'm not sure there is any ambiguity with these attributes. But it doesn't hurt to have a note about the intent in a place like this.

Reply:
I don't think there's any ambiguity either, this text is merely saying what we are aiming for.